### ISSUE TITLE
Provide better expand/collapse animations

### ISSUE THREAD

**Issue by WonderCsabo** [bignerdranch/expandable-recycler-view#136]
Currently the library just calls up to `notifyItemRemoved()` when collapsing an item and `notifyItemInserted()` when expanding an item. With the `DefaultItemAnimator` (which is `RecyclerView` uses by default, hence most of the projects including your examples do), this will result in simply fade in/out animations when expanding the items and collapsing the items.

However, this does not really feel a "real" expansion operation. The expansion would be better represented by a shift animation. It would be nice if the lib would provide an `ItemAnimator` which would animate the expand/collapse action in this manner.

**Comment by dgreenhalgh** [comment]
I agree - this is something we've discussed for a future release. If you end up making this improvement in your project, please feel free to contribute it

**Comment by dlsdnd345** [comment]
i am not arrow rotation animation.. 2.1.1 version

1.1.3 

faqExpandableAdapter.setCustomParentAnimationViewId(R.id.imgArrow);
faqExpandableAdapter.setParentClickableViewAnimationDefaultDuration();
faqExpandableAdapter.setParentAndIconExpandOnClick(true);

Let me know how to use
